### DISCUSSION TITLE
Have void close both the original transaction and the voiding one

### DISCUSSION THREAD

**Discussion by einhverfr** [ledgersmb/LedgerSMB#5863] [Design]
Previous discussion at https://sourceforge.net/p/ledger-smb/feature-requests/197/

Right now you have to do  $0 payment to close them both

Depends on #7460

**Comment by ehuelsmann** [comment]
With open item tracking, the possibility is introduced to close both invoices with a GL transaction instead of a zero payment:

The GL transaction posts it's debit and credit both on the AR/AP account where one of the lines is allocated to the voiding invoice and the other to the voided one. This removes the need for the intermediate cash account.

**Comment by ehuelsmann** [comment]
This can not use a payment transaction, because there is no cashflow on a cash account. That requires attention in the UI: what does the "payment" line look like?

**Comment by ehuelsmann** [comment]
This wants a batch with two transactions: the "payment" and the voiding invoice.

**Reply by ehuelsmann** [reply]
This is similar to when payments get entered on the invoice entry screen.
